docs/user/howto/lintchart.md

```diff
@@ -12,7 +12,7 @@ If the linter encounters things that will cause the chart to fail installation,
 
 For example, running it against one of our test charts with Hypper annotations should produce a warning due to a missing icon:
 
-```shell
+```terminal
```

Comment:
see comment above

Reply:
resolving, let's leave the other conversation.